### PR TITLE
Add design page to cra template

### PR DESCRIPTION
**Pull request by abarkellumich**

### CODE REVIEW

File: template/src/components/Designs.js [hunk 1]
```diff
@@ -0,0 +1,175 @@
+import React, { Component } from 'react';
```
Comment (rlreamy) [inline template/src/components/Designs.js:1]: Why is this in the app template code? This seems to be specific to the table library we used in spatial-viewer, and I didn't think we were going to make this a standard table library.

File: template.json [hunk 2]
```diff
@@ -13,8 +13,9 @@
     "reactstrap": "7.1.0",
     "redux": "4.0.1",
     "redux-thunk": "2.3.0",
-    "kpmp-common-styles":"^1.0.4",
-    "kpmp-common-components": "^1.1.2"
+    "kpmp-common-styles":"^1.0.6",
+    "kpmp-common-components": "^1.1.2",
+    "react-table-filter": "^2.0.2"
```
Comment (rlreamy) [inline template.json:18]: Do we want to include this in the app template?

File: template/src/components/Designs.js (hunk 1 above)
Comment (abarkellumich) [reply template/src/components/Designs.js:1]: Needed something for a table library, happy to swap with whatever we end up fully going with

File: template.json (hunk 2 above)
Comment (abarkellumich) [reply template.json:18]: Same as above, needed something for a table, this would be swappable with whatever we end up going with

**Review by rlreamy — APPROVED** [review]

**Pull request merged**